scripts/us_bts/latch/constants.py

```diff
@@ -0,0 +1,119 @@
+    "est_vmiles": "VehicleMilesTraveled",
+    "vmiles": "VehicleMilesTraveled",
+}
+ACS_LT_MOR = {0: '', 1: '_MarginOfErrorMoreThanACSSurvey'}
+INCOMPLETE_ACS = {0: '', 1: '_IncompleteACSSurvey'}
+URBAN = {1: "Urban", 2: "SemiUrban", 3: "Rural"}
+
+COMMON_COLS = [
+    "ptrp_1mem_0veh", "ptrp_1mem_1veh", "ptrp_1mem_2veh", "ptrp_1mem_3veh",
+    "ptrp_1mem_4veh", "ptrp_2mem_0veh", "ptrp_2mem_1veh", "ptrp_2mem_2veh",
+    "ptrp_2mem_3veh", "ptrp_2mem_4veh", "ptrp_3mem_0veh", "ptrp_3mem_1veh",
+    "ptrp_3mem_2veh", "ptrp_3mem_3veh", "ptrp_3mem_4veh", "ptrp_4mem_0veh",
+    "ptrp_4mem_1veh", "ptrp_4mem_2veh", "ptrp_4mem_3veh", "ptrp_4mem_4veh",
+    "pmiles_1mem_0veh", "pmiles_1mem_1veh", "pmiles_1mem_2veh",
+    "pmiles_1mem_3veh", "pmiles_1mem_4veh", "pmiles_2mem_0veh",
+    "pmiles_2mem_1veh", "pmiles_2mem_2veh", "pmiles_2mem_3veh",
+    "pmiles_2mem_4veh", "pmiles_3mem_0veh", "pmiles_3mem_1veh",
+    "pmiles_3mem_2veh", "pmiles_3mem_3veh", "pmiles_3mem_4veh",
+    "pmiles_4mem_0veh", "pmiles_4mem_1veh", "pmiles_4mem_2veh",
+    "pmiles_4mem_3veh", "pmiles_4mem_4veh", "vtrp_1mem_0veh", "vtrp_1mem_1veh",
+    "vtrp_1mem_2veh", "vtrp_1mem_3veh", "vtrp_1mem_4veh", "vtrp_2mem_0veh",
+    "vtrp_2mem_1veh", "vtrp_2mem_2veh", "vtrp_2mem_3veh", "vtrp_2mem_4veh",
+    "vtrp_3mem_0veh", "vtrp_3mem_1veh", "vtrp_3mem_2veh", "vtrp_3mem_3veh",
+    "vtrp_3mem_4veh", "vtrp_4mem_0veh", "vtrp_4mem_1veh", "vtrp_4mem_2veh",
+    "vtrp_4mem_3veh", "vtrp_4mem_4veh", "vmiles_1mem_0veh", "vmiles_1mem_1veh",
+    "vmiles_1mem_2veh", "vmiles_1mem_3veh", "vmiles_1mem_4veh",
+    "vmiles_2mem_0veh", "vmiles_2mem_1veh", "vmiles_2mem_2veh",
+    "vmiles_2mem_3veh", "vmiles_2mem_4veh", "vmiles_3mem_0veh",
+    "vmiles_3mem_1veh", "vmiles_3mem_2veh", "vmiles_3mem_3veh",
+    "vmiles_3mem_4veh", "vmiles_4mem_0veh", "vmiles_4mem_1veh",
+    "vmiles_4mem_2veh", "vmiles_4mem_3veh", "vmiles_4mem_4veh"
+]
+
+ADDITIONAL_2009_FILE_COLS = [
+    "ptrp_5mem_0veh", "ptrp_5mem_1veh", "ptrp_5mem_2veh", "ptrp_5mem_3veh",
+    "ptrp_5mem_4veh", "pmiles_5mem_0veh", "pmiles_5mem_1veh",
+    "pmiles_5mem_2veh", "pmiles_5mem_3veh", "pmiles_5mem_4veh",
+    "vtrp_5mem_0veh", "vtrp_5mem_1veh", "vtrp_5mem_2veh", "vtrp_5mem_3veh",
+    "vtrp_5mem_4veh", "vmiles_5mem_0veh", "vmiles_5mem_1veh",
+    "vmiles_5mem_2veh", "vmiles_5mem_3veh", "vmiles_5mem_4veh"
+]
+
+CONF_2009_FILE = {
+    "input_file_delimiter": "\t",
+    "basic_cols": ["geoid", "urban_group"],
+    "pop_cols": [
+        "est_pmiles2007_11", "est_ptrp2007_11", "est_vmiles2007_11",
+        "est_vtrp2007_11"
+    ],
+    "extra_cols": [COMMON_COLS, ADDITIONAL_2009_FILE_COLS],
+    "year": 2009
+}
+CONF_2017_FILE = {
+    "input_file_delimiter": ",",
+    "basic_cols": [
+        "geocode", "urban_group", "flag_acs_lt_moe", "flag_incomplete_acs",
+        "flag_manhattan_trt"
+    ],
+    "pop_cols": ["est_pmiles", "est_ptrp", "est_vmiles", "est_vtrp"],
+    "extra_cols": [COMMON_COLS],
```

Comment:
Can you look into why is this list of lists.

Reply:
Created simple list and code is also updated accordinly.